test/interop/python/PREDIFF

```diff
@@ -17,3 +17,5 @@ sed '/^[ ]*$/d' $2 > $2.tmp
 # or just multiple copies of it)
 sed '/^ld: warning/d' $2.tmp > $2
 rm $2.tmp
+# Remove the __init__.py file to prevent a warning.
+rm lib/__init__.py
```

Comment:
The warning you're talking about is due to every test creating `__init__.py`, right?  Meaning that putting it in a cleanfiles wouldn't help because those all get run at the beginning of the test run . . .

I'd be tempted to add a flag to prevent the generation of the `__init__.py` file and through it for these directories, but I'm not sure how useful that would be to users . . .

Reply:
I can also give each individual test its own cleanfiles, but that's a lot of cleanfiles files.

Reply:
Do we want to add a flag to skip generating the init, like `--library-python-no-init`?

Reply:
I don't think the cleanfiles strategy would work anyways, because every subsequent test would have the init file from the previous test.  I slightly prefer a flag over explicitly removing the file to prevent the warning, though it does mean we're doing even less testing of the "normal" user behavior.  But I won't be upset if you choose not to.

Reply:
I think I would rather just rely on the `PREDIFF` until such time as we choose to "modernize" all the old tests, rather than add something to the compiler.